source/elm-live.js

```diff
@@ -29,10 +30,22 @@ module.exports = (argv, options) => {
 
   // Display help
   if (args.help) {
-    spawnSync('man', [
+    const manProcess = spawnSync('man', [
       path.resolve(__dirname, '../manpages/elm-live.1'),
     ], { stdio: 'inherit' });
```

Comment:
This causes the error to be output to the console, followed by the help text.  I think you'll need to set the [`stdio`](https://nodejs.org/api/child_process.html#child_process_options_stdio) option to ignore the error stream (and, I suppose, then write the error stream to `process.stderr` if the error isn't `'ENOENT'`).